docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  app:
+    build: './docker'
+    ports:
+      - "8089:80"
+    volumes:
+      - ./gtg_dev_dir/galaxy_tool_repository:/var/www/html/sites/default/files/galaxy_tool_repository
+      - ./gtg_dev_dir/shed_tools:/var/www/html/sites/default/files/shed_tools
+      - ./galaxy_tool_generator:/var/www/html/sites/all/modules/galaxy_tool_generator/galaxy_tool_generator
+      - ./galaxy_tool_generator_ui:/var/www/html/sites/all/modules/galaxy_tool_generator/galaxy_tool_generator_ui
+    command:
+      - sleep 30
+      - drush en -y galaxy_tool_generator galaxy_tool_generator_ui
+  galaxy:
+    image: 'bgruening/galaxy-stable:17.09'
+    environment:
+      - ENABLE_TTS_INSTALL=True
+      - GALAXY_CONFIG_BRAND=GTG
+    volumes:
+      - ./gtg_dev_dir/shed_tools:/export/shed_tools
```

Comment:
For clarity would it make more sense to define explicit seperate volumes as their own service if they are used by both services?

Then the above line would be `shed_tools`.  

Or do you think thats not necessarily more helpful?

Reply:
It'll make it look better (and I like that) but I wasn't sure how to do it so I used the good old way

Reply:
https://stackoverflow.com/questions/36387032/how-to-set-a-path-on-host-for-a-named-volume-in-docker-compose-yml/49920624#49920624

apparently you specify the local volume driver, and then you can specify the path of a named volume.